src/messaging/CHIPReliableMessageContext.h

```diff
@@ -0,0 +1,112 @@
+/*
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines the reliable message context for the CHIP Message
+ *      Layer. The context is one-on-one relationship with a chip session.
+ */
+
+#ifndef CHIP_RELIABLE_MESSAGE_CONTEXT_H
+#define CHIP_RELIABLE_MESSAGE_CONTEXT_H
+
+#include <stdint.h>
+#include <string.h>
+
+#include <core/CHIPError.h>
+#include <core/ReferenceCounted.h>
+#include <inet/InetLayer.h>
+#include <messaging/CHIPReliableMessageConfig.h>
```

Comment:
config headers should be first includes from another header

Reply:
done